ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_loss_functions.py

```diff
@@ -502,7 +502,7 @@ def test_torch_kl_div(
 @handle_frontend_test(
     fn_tree="torch.nn.functional.l1_loss",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
```

Comment:
The best practice for our tests is to use `get_dtypes("valid")` and define the supported dtypes using `@with_supported_dtypes`/`@with_unsupported_dtypes` on the actual function.

Reply:
I have made the changes, but doesn't it bloat the codebase if we add it to every function. I mean similar to the decision of not adding `documentation` to the functions shouldn't we rely on users to check supported dtypes by using the frontends documentation, rather than adding this to every function

Reply:
Oh right, hadn't thought of that.